releasenotes/notes/prepare-1.2.3-1386aaab85a3fde0.yaml

```diff
@@ -0,0 +1,6 @@
+---
+prelude: >
+    Qiskit 1.2.3 is a patch release fixing a problem in QPY that could cause files generated by
```

Comment:
```suggestion
    Qiskit v1.2.3 is a patch release fixing a problem in QPY that could cause files generated by
```

Reply:
Ah, yeah this is new guidance we got with version numbers. It'd be helpful to use this in other places you have a version string.

Reply:
From the other PR: I'll leave this as-is for this release, since if this is a general editorial style we want to switch to, we'll need to do the entire 1.2 series of release notes in one go (and probably all the other series of release notes too) to avoid intra-page fragmentation of style.